frontend/app/models/recurrence-options.js

```diff
@@ -0,0 +1,8 @@
+import Ember from 'ember';
+
+export default [
+  Ember.Object.create({ value: 0, label: "Daily" }),
```

Comment:
I might do that in a separate PR if you don't mind. this one is getting too big!